src/main/Synthesizer.ts

```diff
@@ -368,11 +368,16 @@ export default class Synthesizer implements ISynthesizer {
 		return this.flushFramesAsync();
 	}
 
-	public loadSFont(bin: ArrayBuffer) {
+	public loadSFont(bin: ArrayBuffer | Uint8Array) {
 		this.ensureInitialized();
 
 		const name = makeRandomFileName('sfont', '.sf2');
-		const ub = new Uint8Array(bin);
+		let ub;
+		if(bin.constructor.name === 'Uint8Array'){
+			ub = bin;
```

Comment:
Nice work!
But, I think it's better to accept `ArrayBufferView` rather than `Uint8Array`.
To achieve this, the implementation will be:

```suggestion
		if ('buffer' in bin) {
			ub = new Uint8Array(bin.buffer, bin.byteOffset, bin.byteLength);
```

Reply:
But doesn't that still create a new typed array?

Reply:
No, `new Uint8Array(buffer)` doesn't copy ArrayBuffer, but simply refers the buffer.

Reply:
Hmm I didn't know that. In that case I think my PR is rather pointless.

Reply:
OK, I'll close this. Thanks anyway.